css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
These selectors should be in reverse order to demonstrate how specificity will override source order.